src/sections/Why/History.tsx

```diff
@@ -10,16 +10,13 @@ export const History = () => (
       <Text as="span" variant="emphasis">
         History:
       </Text>{' '}
-      Software engineer{' '}
-      <Link href="https://twitter.com/tmeasday">Tom Coleman</Link> introduced
+      Software engineer <Link href="https://twitter.com/tmeasday">Tom Coleman</Link> introduced
       Component Driven in{' '}
-      <Link href="https://blog.hichroma.com/component-driven-development-ce1109d56c8e">
-        2017
-      </Link>{' '}
-      to describe the shift in UI development toward component architectures.
-      The idea of modular software has many parallels in movements such as
-      microservices and containerization. Historical precedents also include
-      lean manufacturing and mass manufacturing circa early 20th century.
+      <Link href="https://blog.hichroma.com/component-driven-development-ce1109d56c8e">2017</Link>{' '}
+      to describe the shift in UI development toward component architectures and processes. The idea
+      of modular UI has many parallels in software movements such as microservices and
```

Comment:
I don't think it made sense to say "modular software has parallels" and then mention two types of software. Also modular software has been around literally[1] forever.

(Modular UIs have been around forever too, but less prominently in the web world).

[1] Literally not literally